components/atom/image/src/index.js

```diff
@@ -29,6 +29,8 @@ const AtomImage = ({
   errorText,
   onError,
   onLoad,
+  sources,
```

Comment:
`sources = [],` and remove the default props section. react-docs parses this value too.